pyramid_oereb/contrib/stats/scripts/templates/views.sql.mako

```diff
@@ -0,0 +1,95 @@
+
+
+/*
```

Comment:
Should we remove this example (now that we have real views below)

Reply:
done